.github/workflows/integration-test-setup/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: Cluster Setup
+description: Cluster setup for integration test
+inputs:
+  kubernetes-version:
+    description: Name to use for the workflow
+    required: true
+  github-token:
+    description: GITHUB_TOKEN from the calling workflow
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+      - name: setup golang
```

Comment:
can we include:

```
    - name: checkout
      uses: actions/checkout@v2
      with:
        fetch-depth: 0
```


too?

Reply:
no, because it searches for the file(action.yaml) so if we include this in composite, action will not able to find composite yaml